test/snippet/core/algorithm/configuration_get.cpp

```diff
@@ -12,11 +11,11 @@ int main()
 
     seqan3::configuration my_cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-1},
                                                                              seqan3::gap_open_score{-10}}} |
-                                   seqan3::align_cfg::band{seqan3::static_band{seqan3::lower_bound{-4},
-                                                                               seqan3::upper_bound{4}}};
+                                   seqan3::align_cfg::band_fixed_size{seqan3::align_cfg::lower_diagonal{-4},
+                                                                      seqan3::align_cfg::upper_diagonal{4}};
     // my_cfg is now of type configuration<gap, band>
 
-    seqan3::debug_stream << get<1>(my_cfg).value.lower_bound                          << '\n';  // prints -4
-    seqan3::debug_stream << get<seqan3::align_cfg::band>(my_cfg).value.upper_bound    << '\n';  // prints 4
-    seqan3::debug_stream << get<seqan3::align_cfg::gap>(my_cfg).value.get_gap_score() << '\n';  // prints -1
+    seqan3::debug_stream << get<1>(my_cfg).lower_diagonal                                  << '\n';  // prints -4
+    seqan3::debug_stream << get<seqan3::align_cfg::band_fixed_size>(my_cfg).upper_diagonal << '\n';  // prints 4
```

Comment:
Only for this? Why not use `get<1>(my_cfg).lower_diagonal.get()`?

Reply:
Because the strong type is a detail class and the user does not know about the get interface.